de.dlr.sc.virsat.model.ui/src/de/dlr/sc/virsat/model/ui/provider/ActiveConceptsLabelProvider.java

```diff
@@ -25,10 +25,24 @@
 	public String getText(Object element) {
 		if (element instanceof ActiveConceptConfigurationElement) {
 			ActiveConceptConfigurationElement acce = (ActiveConceptConfigurationElement) element;
-			return acce.getId() + " [" + acce.getVersion() + "]";
+			String acceDisplayName = acce.getDisplayName();
+			
+			if (acceDisplayName == null) {
+				return acce.getId() + " [" + acce.getVersion() + "]";
+				
+			} else {
+				return acceDisplayName + " – " + acce.getId() + " [" + acce.getVersion() + "]";
+			}
+			
 		} else if (element instanceof Concept) {
 			Concept concept = (Concept) element;
-			return concept.getName() + " [" + concept.getVersion() + "]";
+
+			if (concept.getDisplayName() == null) {
+				return concept.getName() + " [" + concept.getVersion() + "]";

```

Comment:
Same as above, I think we can extract the `concept.getName() + " [" + concept.getVersion() + "]"` from the if-else so we dont have redudant logic.

Reply:
Agree.